integration-tests/contracts/ValueCalls.sol

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: MIT
+// @unsupported: evm
```

Comment:
Why is this unsupported in the EVM?

Reply:
Just came in from whatever I copy-pasted -- will delete.